charts/s3-chart/templates/deployment.yaml

```diff
@@ -69,7 +69,7 @@ spec:
             fieldRef:
               fieldPath: metadata.namespace
         - name: AWS_REGION
-          value: {{ .Values.aws.region }}
+          value: {{ required "A valid aws-region required!" .Values.aws.region }}
```

Comment:
We should strive to have minimum changes in 3rd party charts, so I would get rid of this